webapp/brilla-frontend/components/videoplayer.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
+import React, { FC, useState, useEffect } from "react";
+import dynamic from "next/dynamic";
+// import Spinner from "./ui/spinner";
+import Shimmer from "./ui/shimmer";
+
+interface VideoPlayerProps {
+  url: string;
+}
+
+const ReactPlayer = dynamic(() => import("react-player/youtube"), {
+  ssr: false,
+});
+
+const VideoPlayer: FC<VideoPlayerProps> = ({ url }) => {
+  const [loading, setLoading] = useState(true);
+
+  const handleReady = () => {
+    setLoading(false);
+  };
+
+  return (
+    <div className="flex flex-col md:flex-row md:max-w-[45vw] h-[28vh] relative">
+      {/* {loading && <Spinner />} */}
+      {loading && <Shimmer />}
+      <ReactPlayer
```

Comment:
just to check, this works for live videos as well right?

Reply:
Yes please it works for live videos